system_launch/launch/system.launch.xml

```diff
@@ -2,21 +2,22 @@
 <launch>
   <arg name="run_mode" default="online" doc="options: online, planning_simulation"/>
 
-  <group ns="/system">
+  <group>
     <!-- State Monitor -->
-    <include file="$(find autoware_state_monitor)/launch/autoware_state_monitor.launch">
-      <arg name="config_file" if="$(eval run_mode == 'online')" value="$(find autoware_state_monitor)/config/autoware_state_monitor.yaml" />
-      <arg name="config_file" if="$(eval run_mode == 'planning_simulation')" value="$(find autoware_state_monitor)/config/autoware_state_monitor.planning_simulation.yaml" />
-    </include>
+    <push-ros-namespace namespace="/system"/>
+    <!-- <include file="$(find-pkg-share autoware_state_monitor)/launch/autoware_state_monitor.launch.xml">
+      <arg name="config_file" if="$(eval run_mode == 'online')" value="$(find-pkg-share autoware_state_monitor)/config/autoware_state_monitor.yaml" />
+      <arg name="config_file" if="$(eval run_mode == 'planning_simulation')" value="$(find-pkg-share autoware_state_monitor)/config/autoware_state_monitor.planning_simulation.yaml" />
+    </include> -->
 
     <!-- Error Monitor -->
-    <include file="$(find autoware_error_monitor)/launch/autoware_error_monitor.launch">
-      <arg name="config_file" value="$(find autoware_error_monitor)/config/autoware_error_monitor.yaml" />
+    <include file="$(find-pkg-share autoware_error_monitor)/launch/autoware_error_monitor.launch.xml">
+      <arg name="config_file" value="$(find-pkg-share autoware_error_monitor)/config/autoware_error_monitor.yaml" />
```

Comment:
please replace with `autoware_error_monitor.param.yaml`

Reply:
Fixed in 46aa1dd